### PR TITLE
feat(profile) profile management when deploying an organization

### PR DESCRIPTION
**Pull request by rbioteau**

### CODE REVIEW

File: md/organization-management-in-bonita-bpm-studio.md [hunk 1]
```diff
@@ -74,11 +74,24 @@ Bonita Portal, which could have an impact on deployed processes.
...
 Bonita Studio. This user must be in the organization you have
 selected.
-4. Click **Publish**.
+4. Click **Deploy**.
+
+## Profile management
+
+In order to log in to bonita UIs (Portal, applications...), a user must have at least on profile (User, Adminstrator...etc).
+
+**_In Community_**, only 3 provided profiles are supported (User, Adminstrator and Process Manager). When deploying an organization from the Studio, each user is automatically mapped to all those profiles.
```
Comment (alachambre) [inline md/organization-management-in-bonita-bpm-studio.md]: Wrong, Process Manager is available in teamwork

File: md/organization-management-in-bonita-bpm-studio.md [hunk 2]
```diff
@@ -74,11 +74,24 @@ Bonita Portal, which could have an impact on deployed processes.
...
+
+**_In Teamwork edition_**, those 3 provided profiles can edited in the Studio using an XML editor (Go to Organization > Profiles > Open).
```
Comment (alachambre) [inline md/organization-management-in-bonita-bpm-studio.md]: can be edited

**Review by alachambre — CHANGES_REQUESTED** [review]

File: md/organization-management-in-bonita-bpm-studio.md [hunk 3]
```diff
@@ -74,11 +74,24 @@ Bonita Portal, which could have an impact on deployed processes.
...
+3. Specify the username of the user that will be
 used to log in to Bonita Portal when a process is started from
 Bonita Studio. This user must be in the organization you have
 selected.
-4. Click **Publish**.
+4. Click **Deploy**.
+
+## Profile management
+
+In order to log in to bonita UIs (Portal, applications...), a user must have at least on profile (User, Adminstrator...etc).
```
Comment (NathalieC) [inline md/organization-management-in-bonita-bpm-studio.md]: .. at least one profile

File: md/organization-management-in-bonita-bpm-studio.md (hunk 2 above)
Comment (NathalieC) [reply md/organization-management-in-bonita-bpm-studio.md]: Is role "member" automatically mapped to all profiles by default?

File: md/organization-management-in-bonita-bpm-studio.md [hunk 4]
```diff
@@ -74,11 +74,24 @@ Bonita Portal, which could have an impact on deployed processes.
...
+
+**_In Community_**, only 3 provided profiles are supported (User, Adminstrator and Process Manager). When deploying an organization from the Studio, each user is automatically mapped to all those profiles.
+
+**_In Teamwork edition_**, those 3 provided profiles can edited in the Studio using an XML editor (Go to Organization > Profiles > Open).
+
+**_In Efficiency and Performance editions_**, profiles can be defined in the Studio using the [Profile Editor](profileCreation.md).
+
```
Comment (NathalieC) [inline md/organization-management-in-bonita-bpm-studio.md]: "By default, all users of ACME are mapped to the role _member_, and the role _memeber_ is automatically mapped to all default profiles.
=> also to custom profiles, or only default ones?
Therefore, for testing purposes, you can use any ACME user to login to Bonita Portal and applications.

File: md/organization-management-in-bonita-bpm-studio.md [hunk 5]
```diff
@@ -74,11 +74,24 @@ Bonita Portal, which could have an impact on deployed processes.
...
+When deploying an organization make sure to synchronize those profiles with the organization. You will be notified when a user is not mapped to any profile while deploying an organization.
```
Comment (NathalieC) [inline md/organization-management-in-bonita-bpm-studio.md]: "When creating or importing other organizations, you need to make sure that all users are mapped to at least one profile, through a group, a role, a membership, or as a user, to grant them access to Bonita Portal or applications."

Comment (NathalieC) [reply md/organization-management-in-bonita-bpm-studio.md]: IMHO, there is no need to say there are notifications. 
It's super good they are implemented, but they don't need to be promoted in the doc that just needs to say what to do and how to do it.

File: md/organization-management-in-bonita-bpm-studio.md (hunk 4 above)
Comment (rbioteau) [reply md/organization-management-in-bonita-bpm-studio.md]: In community only User and Adminstrator profiles exist.
Each users of the an organization (ACME or other) are mapped to those 2 profiles using a user mapping.

**Pull request merged**